test/e2e/fixture/app/actions.go

```diff
@@ -80,6 +80,11 @@ func (a *Actions) CreateFromFile(handler func(app *Application)) *Actions {
 			NamePrefix: a.context.namePrefix,
 		}
 	}
+	if a.context.nameSuffix != "" {
```

Comment:
what happens if you have both suffix and prefix - doesn't this replace the prefix?

Reply:
yeah that's a good point, I'll rework this bit and update the tests

Reply:
I felt like wasn’t the only place. Need to check the other code.

Reply:
I've double checked, and I don't see any potential issues for cases where nameprefix and namesuffix are both set. Did you see anything else in particular that you were concerned about?